packages/firestore/package.json

```diff
@@ -97,8 +97,11 @@
     "@firebase/auth": "0.20.4",
     "@rollup/plugin-alias": "3.1.9",
     "@rollup/plugin-json": "4.1.0",
+    "@types/chai": "^4.3.1",
     "@types/eslint": "7.29.0",
     "@types/json-stable-stringify": "1.0.34",
+    "chai": "^4.3.6",
+    "chai-exclude": "^2.1.0",
```

Comment:
We usually pin dependencies to the specific version, so this should be `"chai-exclude": 2.1.0`

Reply:
Fixed.